docs/book/src/reference/good-practices.md

```diff
@@ -0,0 +1,24 @@
+# Good Practices
+
+## What is "Reconciliation" in Operators?
```

Comment:
```suggestion
## What is "Reconciliation" in Operators?
```

We have no Operators. We have projects scaffolded with Kubebuilder, which might or not be Operators. Usually, that is the common goal. So, I am not sure about this question/topic.